lib/test_after_commit.rb

```diff
@@ -1,6 +1,33 @@
 require 'test_after_commit/version'
 
 module TestAfterCommit
+  @enabled_by_default = true
```

Comment:
That works. I was just trying to hide the internal storage but that was probably overkill. I'll get the change up.